eslint-config-landr/index.js

```diff
@@ -100,6 +101,13 @@ module.exports = {
                 '@typescript-eslint/no-inferrable-types': 'error',
                 '@typescript-eslint/type-annotation-spacing': 'error',
                 '@typescript-eslint/no-use-before-define': 'error',
+                '@typescript-eslint/no-unused-vars': [
```

Comment:
based on the docs, you need to disable the base rule first
`"no-unused-vars": "off",`

Reply:
Thanks, good catch!
fixed